semgrep-core/bin/Main.ml

```diff
@@ -347,6 +348,24 @@ let timeout_function lang = fun f ->
   then f ()
   else Common.timeout_function ~verbose:!verbose timeout f
 
+(* from https://discuss.ocaml.org/t/todays-trick-memory-limits-with-gc-alarms/4431 *)
+let run_with_memory_limit limit f =
```

Comment:
Does this mean semgrep-core will use 4GB split among all the jobs or each one can use 4GB?